guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,31 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Pulpcore and plugins upgraded to Pulpcore 3.28.
+Repositories now use pulp_deb 3.0.0.
```

Comment:
I'd explain this a bit further. For example, Katello 4.9 used Pulp 3.22 so you can mention that, as well as some of the major features since Pulp 3.22.